contributing/projet-management.md

```diff
@@ -0,0 +1,105 @@
+# Project management guidelines
+
+|IMPORTANT: The project management guidelines are only applicable to AMP Stories at this stage.|
+| :--- |
+
+# Project boards
+
+In addition to [Milestones](https://github.com/ampproject/amp-wp/milestones), which are used to manage releases, project boards are used to manage issue statuses.
+
+### Definition project board
+
+The [Definition](https://github.com/ampproject/amp-wp/projects/14) project board covers the pipeline which issues go through in preparation for execution. The board contains the following columns:
+
+* Revisit Later
+* Prioritization
+* Acceptance Criteria
+* Implementation Brief / Estimate
```

Comment:
How does the estimation in this step differ from the last? Is it that in the event an Implementation Brief is not needed we would Estimate here?